autobot-backend/llc/tests/test_company_ceo_15770.py

```
@@ -0,0 +1,549 @@
+    company_id = await _seed_company_without_ceo(session_factory)
+    await _seed_owner(session_factory, company_id)
+    worker = await _seed_agent(session_factory, company_id, "worker-loop")
+
+    async with session_factory() as session:
+        ceo_row = await CompanyCEOService().provision_default(session, company_id)
+        ceo_agent = ceo_row.holder_agent_id
+        await session.commit()
+
+    service = ReportingLineService()
+    async with session_factory() as session:
+        with pytest.raises(ValueError):
```

Ruff (0.16.3)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)